cypress/integration/Form_spec.js

```diff
@@ -0,0 +1,55 @@
+describe('Form', () => {
+  beforeEach(() => {
+    //intercept
+    cy.visit('http://localhost:3000')
+      .get('.add-tree').click()
+  })
+  it('Should have a controlled input field for a tree name whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="name"]').type('tree name')
+      .get('form input[name="name"]').should('have.value', 'tree name')
+  })
+  it('Should have a controlled input field for a tree scientific name whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="scientificName"]').type('tree scientific name')
+      .get('form input[name="scientificName"]').should('have.value', 'tree scientific name')
+  })
+  it('Should have a controlled input field for a new tree region whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="region"]').type('tree region')
+      .get('form input[name="region"]').should('have.value', 'tree region')
+  })
+  it('Should have a controlled input field for a new tree average height whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="averageHeight"]').type('tree average height')
+      .get('form input[name="averageHeight"]').should('have.value', 'tree average height')
+  })
+  it('Should have a controlled input field for a new tree lifespan whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="lifespan"]').type('tree lifespan')
+      .get('form input[name="lifespan"]').should('have.value', 'tree lifespan')
+  })
+  it('Should have a controlled input field for a new tree fun fact whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="fact"]').type('tree fact')
+      .get('form input[name="fact"]').should('have.value', 'tree fact')
+  })
+  it('Should have a controlled input field for a new tree image whose value reflects the data typed into the form', () => {
+    cy.get('form input[name="image"]').type('tree image link')
+      .get('form input[name="image"]').should('have.value', 'tree image link')
+  })
+  it('Should be able to create a new tree when the Add Tree button is clicked', () => {
```

Comment:
I agree that we should use an intercept for testing, otherwise I think it could get really messy with adding items to our API  for every test.